WikiWatershed/model-my-watershed
# Calculate and display modification areas

* Calculate area of modification geojson and replace placeholder value in
modification summary with actual value.

To test:

- Create a project.
- Draw some modifications.
- Check that areas are computed in the modifications summary.

![selection_040](https://cloud.githubusercontent.com/assets/1042475/7923566/2e679f4a-0882-11e5-8889-31af16040ea6.png)

Connects to #236
